deploy/bundle/manifests/devworkspace-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,373 @@
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterrolebindings
+          - clusterroles
+          - rolebindings
+          - roles
+          verbs:
+          - create
+          - get
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - '*'
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes/custom-host
+          verbs:
+          - create
+        - apiGroups:
+          - workspace.devfile.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: devworkspace-controller-serviceaccount
+      deployments:
+      - name: devworkspace-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app.kubernetes.io/name: devworkspace-controller
+              app.kubernetes.io/part-of: devworkspace-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app.kubernetes.io/name: devworkspace-controller
+                app.kubernetes.io/part-of: devworkspace-operator
+            spec:
+              containers:
+              - args:
+                - /usr/local/bin/devworkspace-controller
+                - --enable-leader-election
+                env:
+                - name: DEVELOPMENT_MODE
+                  value: "true"
+                - name: RELATED_IMAGE_devworkspace_webhook_server
+                  value: quay.io/devfile/devworkspace-controller:next
+                - name: RELATED_IMAGE_project_clone
+                  value: quay.io/devfile/project-clone:next
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: devworkspace-operator
+                - name: CONTROLLER_SERVICE_ACCOUNT_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: spec.serviceAccountName
+                - name: MAX_CONCURRENT_RECONCILES
+                  value: "1"
+                - name: WEBHOOKS_SERVER_MEMORY_LIMIT
+                  value: 300Mi
+                - name: WEBHOOKS_SERVER_MEMORY_REQUEST
+                  value: 20Mi
+                - name: WEBHOOKS_SERVER_CPU_LIMIT
+                  value: 200m
+                - name: WEBHOOKS_SERVER_CPU_REQUEST
+                  value: 100m
+                - name: RELATED_IMAGE_plugin_redhat_developer_web_terminal_4_5_0
```

Comment:
We technically don't, but it might still be useful to have for now if we're testing WTO with the old v1alpha1 devworkspace (no idea if it actually works, though...). 

Removing the env var here would also probably mean removing the internal registry and related logic, which maybe makes sense as a separate PR.